src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -316,6 +346,22 @@ private void initShuffleboard() {
 					.addDouble("Top FlyWheel Temp", () -> launcherTopMotor.getMotorTemperature());
 		}
 
+		Shuffleboard.getTab("Launcher")
+				.addDouble(
+						"relative encoder", () -> launcherAngleOneMotor.getEncoder().getPosition() * 360);
+		Shuffleboard.getTab("Launcher")
+				.addDouble("relative encoder offset", () -> relativeEncoderStartPosition.orElse(0.0));
+
+		Shuffleboard.getTab("Launcher.add")
+				.addDouble(
+						"relative encoder",
+						() -> Units.rotationsToDegrees(launcherAngleOneMotor.getEncoder().getPosition()));
```

Comment:
Why is this in a differently named tab? Is it redundant with the "relative encoder" entry?

Reply:
fumble